_posts/2022-08-29-hello-evvnt.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title:  "Hello Evvnt"
+date:   2022-08-29 15:50:14 -0400
+categories: engineering
+---
+Welcome to the Evvnt Engineering Blog! This is our hello world introducing Evvnt and what we do. First thing is first, its not a typo! The company name is Evvnt and its pronounced Event. (We don't blame you if you pronounce it like [marwage](https://www.youtube.com/watch?v=3odMTPuzLwY).)
+
+Our Passion: **Powering Experiences to Connect People**
+
+*People Value Social Interaction Through Local Events that Vitalize Communities and Improve Life Quality*
+
+Q: Why haven't I heard of Evvnt?
+A: Today we deliver our software as white labeled. We are on thousands of media sites power event calendars and selling millions of dollars in tickets.
```

Comment:
Today we deliver our software as white labeled service. We are on thousands of media sites powering their event calendars and sell millions of dollars in tickets.